### DISCUSSION TITLE
Kubernetes VM hostname

### DISCUSSION THREAD

**Discussion by LtCmdrScott** [rancher-sandbox/rancher-desktop#2667] [Q&A]
When Rancher Desktop v1.4.1 starts up a Kubernetes cluster, does it change and/or create its own hostname / IP address / virtual adapter when it spins up?  The reason that I ask is that, is in our company configuration, our network ports can be blocked (turned off) if it senses another "computer" (real or virtual) is present on the same interface as the "real machine" and the hostname is NOT the same.

If it does create its own adapter or get its own IP address / hostname, can we configure this?

**Comment by mook-as** [comment]
On macOS only, if you grant it administrative permissions (type in your password when prompted), it will create a bridged network on your machine and request an address via DHCP on your network.  If you disallow that access, then this would not happen; if you've previously granted it, please delete `/etc/sudoers.d/zzzzz-rancher-desktop-lima` on your disk.

If you would like to see that as a separate toggle, I believe the closest issue I can find is https://github.com/rancher-sandbox/rancher-desktop/issues/1227.
